docker/compose/docker-compose.yml

```diff
@@ -79,8 +79,12 @@ services:
     ports:
       - "12345:12345"
     command: segmentstore
+    privileged: true
     environment:
       WAIT_FOR: bookie1:3181,bookie2:3182,bookie3:3183,hdfs:8020
+      ENABLE_HDFS: "false"
+      ENABLE_NFS: "true"
+      NFS_MOUNT: 172.16.39.2:/ifs/pravega
```

Comment:
If this docker-compose file depends on NFS, why is there also a docker-compose-nfs.yml file. I would expect either there are two (one for HDFS and one for NFS). If they both use NFS, why not use a single file?